website/docs/documentationFile.md

```diff
@@ -0,0 +1,43 @@
+---
+title: documentationfile
+---
+
+Enable C# xmlDocumentationFile
+
+# Usage (2) #
+the DocumentationFile is used for adding the [xml comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/) added to functions/ variables, to a dll that has been packed inside a framework or other C# related.
+this can then be referenced inside another project by placing it next to the corresponding dll.
+
+this feature sets the [DocumentationFile](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/output#documentationfile) option inside csproj for each corresponding [configuration](https://premake.github.io/docs/configurations/)
+
+## 1) Default ##
+
+when you put documentationfile inside the project, the following filename/path will be generated:
+```%{targetdir}/%{prj.name}.xml```
+```lua
+documentationfile ""
+```
+
+## 2) Custom Directory ##
+
+when you put the following inside the project the following filename/path will be generated:
```

Comment:
Is there a technical requirement from VS that the documentation must have a name matching that of the project or binary output, or is it a self-impose limitation to make life easier?

Reply:
Yes visual studio cannot recognise the xml file!